tests/time_tests/c++/cli.h

```diff
@@ -0,0 +1,151 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include <gflags/gflags.h>
+#include <iostream>
+
+/// @brief message for help argument
+static const char help_message[] = "Print a usage message";
```

Comment:
make POSIX compatible CLI

Reply:
To read: https://www.gnu.org/prep/standards/html_node/Command_002dLine-Interfaces.html

Reply:
GFlags provides unified way how specify command-line keys (so `--api`, `-api`, `-a`, `--a` is possible ways). Also I didn't find anything about positional arguments

Reply:
Decided to leave it as is